Server/models/Team.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
+const TeamSchema = mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    immutable: true,
+  },
+  timestamps: true,
```

Comment:
I believe this should be 
```javascript
{ timestamps: true }
```
to automatically create the 'createdAt' and 'updatedAt' fields for your documents.

The way it is now I believe it just stores the value true in each document which is not the intended behavior I believe.

Reply:
I think I got this one in a later commit should be good now